ScienceJournal/ActionArea/ActionAreaBarViewController.swift

```diff
@@ -527,8 +537,20 @@ extension ActionArea {
       button.backgroundColor = customTint.secondary
     }
 
+    func setEnabled(_ enabled: Bool, animated: Bool = true) {
+      button.isEnabled = enabled
+      let updatedAlpha: CGFloat = enabled ? 1.0 : BarViewController.Metrics.Bar.disabledAlpha
+      if animated {
+        UIView.animate(withDuration: 0.5) {
```

Comment:
I'd move the animation block up to the VC. It currently controls timings for animations, so it'll be easier to reason about with those in the same place. There's also some logic up there related to enabling/disabling the entire bar/detail era, and the alpha values there and here are likely related.